automation/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:15.04
+
+# Update index
+RUN apt-get update && apt-get -y upgrade
+
+# Install the following utilities (required by poky)
+RUN apt-get install -y build-essential chrpath curl diffstat gcc-multilib gawk git-core libsdl1.2-dev texinfo unzip wget xterm cpio file
```

Comment:
This should end with `&& rm -rf /var/lib/apt/lists/*` in order to save some space in the final image (just makes downloading a bit quicker, and use less storage space)